app/src/main/java/com/beemdevelopment/aegis/ui/views/EntryListView.java

```diff
@@ -144,12 +147,26 @@ public int getSpanSize(int position) {
             @Override
             public void onRefresh() {
                 refresh(false);
+
+                if (_recyclerView.isShown()) {
```

Comment:
Do we still need this check now that we stop the refresher when the activity is not visible?